tools/Localizations/Sources/Localizations/ExportTask.swift

```diff
@@ -68,6 +62,37 @@ struct ExportTask {
         task.waitUntilExit()
     }
 
+    private func getBlockzillaFolder() -> [String] {
```

Comment:
@st3fan Should I move this and getProjects locales to a different file so that it can be used by both this ExportTask and main.swift where I have same code?